README.md

```diff
@@ -4,9 +4,31 @@
 
 Build software only once using [Nix](https://nixos.org/nix/) with the help of [Cachix](https://cachix.org).
 
+This action will configure Cachix and invoke `nix-build`.
+
+## Why do I need this
+
+Because you'd like for your CI to be fast. Let me explain.
+
+Caching on a typical CI doesn't work in favor of Nix.
+
+`/nix/store` is a global storage of everything Nix operates on. These are
+your sources, patches, tarballs, packages, configuration.
+
+Caching `/nix/store` is time consuming as Nix only appends store paths to it.
```

Comment:
> Caching `/nix/store` is time consuming as Nix only appends store paths to it.

This sentence didn't help me at all.

Here's my perspective as a comparison. Needs some polish too.

A directory-based cache requires that downloading a whole store, including the irrelevant parts. `cachix-action` will only fetch what's needed by configuring a Nix binary cache.

When the build is done, `cachix-action` only has to upload the new store paths, rather than syncing the whole store.

Purging paths from a directory-based cache is not feasible because it'd have to be aware of all branches and their respective contents somehow.